opendkim/opendkim.c

```diff
@@ -15470,6 +15474,7 @@ usage(void)
 	                "\t-A          \tauto-restart\n"
 	                "\t-b modes    \tselect operating modes\n"
 	                "\t-c canon    \tcanonicalization to use when signing\n"
+	                "\t-C          \tdo walk SigningTable when loading config\n"
```

Comment:
how about just 
walk the SigningTable when loading config. 
or, 
force walking the SigningTable when loading config.

do walk sounds like default is dont ?

Reply:
The reason of this expression is just I mentioned above (although I didn't explain why I prefer the check is off by default), and meanless now.  So feel free to change the messages and documents.